torchdata/datapipes/iter/__init__.py

```diff
@@ -35,34 +57,52 @@
 ###############################################################################
 # Reference From PyTorch Core
 ###############################################################################
-from torch.utils.data.datapipes.iter import IterableWrapper
 
 __all__ = [
+    "Batcher",
     "BucketBatcher",
     "CSVDictParser",
     "CSVParser",
+    "Collator",
+    "Concater",
     "Cycler",
+    "Demultiplexer",
+    "FileLister",
+    "FileLoader",
+    "Filter",
+    "Forker",
     "GDriveReader",
+    "Grouper",
     "HashChecker",
     "Header",
     "HttpReader",
     "InMemoryCacheHolder",
     "IndexAdder",
     "IoPathFileLister",
     "IoPathFileLoader",
+    "IterDataPipe",
     "IterableWrapper",
     "JsonParser",
     "KeyZipper",
     "LineReader",
+    "Mapper",
+    "Multiplexer",
     "OnDiskCacheHolder",
     "OnlineReader",
     "ParagraphAggregator",
+    "RoutedDecoder",
     "Rows2Columnar",
     "SampleMultiplexer",
+    "Sampler",
     "Saver",
+    "Shuffler",
+    "StreamReader",
     "TarArchiveReader",
+    "UnBatcher",
     "XzFileReader",
     "ZipArchiveReader",
+    "Zipper",
+    "functional_datapipe",
```

Comment:
I would prefer `functional_datapipe` exposed at the parent folder `torchdata.datapipes`, because this decorate can also be attached to the map-style DataPipe.